tests/images/run.sh

```diff
@@ -24,3 +24,7 @@ done
 ./bin/image-classifier tests/images/imagenet/zebra_340.png -image_mode=0to1 -m=shufflenet/model.onnx $@
 
 ./bin/image-classifier tests/images/mnist/*.png -image_mode=0to1 -m=lenet_mnist $@
+
+for png_filename in tests/images/mnist/*.png; do
+  ./bin/image-classifier $png_filename -image_mode=0to1 -m=mnist.onnx $@
```

Comment:
does this model accept batch inference? could you try `tests/images/mnist/*.png`?

Reply:
No. I tried it, and the results are incorrect.

Reply:
Why are the results incorrect? Is there an assertion failure?

Reply:
It is an assertion:  "Batch image inference is not supported by ONNX models."

Reply:
I removed the assert and was able to see where it fails really. Like with ONNX models from the zoo (e.g. resnet50), there is a Reshape to one hard-coded shape (as a result of C2->ONNX conversion). That's why we can't feed any larger batch size to this model: it would be reshape to incorrect size.